### ISSUE TITLE
Change `provider` property into readonly class property?

### ISSUE THREAD

**Issue by pi314** [zdict/zdict#48]
Should `provider` be accessable without creating a dictionary object instance?

**Comment by M157q** [comment]
I'm a little confused about the relationship between the title of this issue and your comment.
Can u describe more specifically?

Are u asking this because of #45 ?
Because it seems need the `provider` property be accessable without creating a dictionary instance.

**Comment by pi314** [comment]
Yes, this issue is a little bit related to #45 (in #45 I need `title` property)
But it's more about design issue, "Should I being able to get `provider` with only the dictionary class?"

**Comment by iblis17** [comment]
I will research on this
